src/mainwindow.cpp

```diff
@@ -197,7 +197,7 @@ MainWindow::MainWindow(QWidget *parent) :
                 ui->cpusAvailableValue->setText(QString::number(data.cpusAvailable));
                 ui->cpuSiblingCoresValue->setText(data.cpuSiblingCores);
                 ui->cpuSiblingThreadsValue->setText(data.cpuSiblingThreads);
-                ui->totalMemoryValue->setText(format.formatByteSize(data.totalMemoryInKiB, 1, KFormat::MetricBinaryDialect));
+                ui->totalMemoryValue->setText(format.formatByteSize(data.totalMemoryInKiB, 1, KFormat::IECBinaryDialect));
```

Comment:
this is still wrong, and actually worse than before ;-)

we want MetricBinaryDialect, but we need to convert totalMemoryInKib to bytes by adding a * 1024 here